src/components/AddEditForm/AddEditForm.vue

```diff
@@ -0,0 +1,182 @@
+<template>
+  <div>
+    <form class="form-container">
+      <input-group
+        label-id="title"
+        label-text="Title"
+      >
+        <template
+          class="input-error-block"
+        >
+          <input
+            v-model="title"
+            v-validate="'required'"
+            class="input"
+            name="title"
+            placeholder="Enter the title"
+          >
+          <span class="error-message">{{ errors.first('title') }}</span>
+        </template>
+      </input-group>
+      <input-group
+        label-id="price"
+        label-text="Price"
+      >
+        <template>
```

Comment:
Is there a specific reason you are wrapping these elements in a `template`?
They should be rendered the same without it. 🤔

Reply:
Thought that it's the same situation as with all components, that sibling component must be wrapped.

Reply:
Nope, default slot accepts anything you put in it, be it a single component or multiple components. 😃